pkg/controllers/secretstore/common.go

```diff
@@ -67,9 +67,7 @@ func reconcile(ctx context.Context, req ctrl.Request, ss esapi.GenericStore, cl
 	cond := NewSecretStoreCondition(esapi.SecretStoreReady, v1.ConditionTrue, esapi.ReasonStoreValid, msgStoreValidated)
 	SetExternalSecretCondition(ss, *cond)
 
-	return ctrl.Result{
-		RequeueAfter: requeueInterval,
-	}, err
+	return ctrl.Result{}, err
```

Comment:
This is actually the good case, since `err=nil` here. We should let the RequeueAfter, otherwise the SecretStore will be requeued only once.

Reply:
😅 omg, why did I purposed that... sry for wasting your time.

Can we keep the vault fix (retry=0)?